src/include/duckdb/planner/filter/conjunction_filter.hpp

```diff
@@ -25,6 +25,7 @@ class ConjunctionFilter : public TableFilter {
 	vector<unique_ptr<TableFilter>> child_filters;
 
 public:
+	virtual void ExtractInFilterValues(vector<Value> &values);
```

Comment:
Given the fact that this is only used in the one place (`table_scan.cpp`) can we perhaps move it out of the filter and keep it close to where it is being used?